src/cljs/ui/splash/page.cljs

```diff
@@ -13,81 +13,156 @@
+                                                     :color      "#757295"
+                                                     :margin     "10px"
+                                                     :padding    "10px"}
+
+                                       :send-button {:width          "325px"
+                                                     :height         "50px"
+                                                     :text-align     :center
+                                                     :font           "normal normal medium 16px/19px Roboto"
+                                                     :text-transform :none
+                                                     :color          "white"
+                                                     :background     "#3428CA"}
+
+                                       :google-button {:width          "325px"
+                                                       :height         "50px"
+                                                       :background     "#F0F0F0 0% 0% no-repeat padding-box"
+                                                       :border         "1px solid #F0F0F0"
+                                                       :border-radius  "8px"
+                                                       :text-transform :none
+                                                       :font           "normal normal medium 16px/27px Roboto"
+                                                       :color          "#757575"}
+
+                                       })))
 
 (defmethod page :route/splash []
-  (let [{:keys [google]}                 (<sub [::subs/config])
-        {:keys [query]}                  (<sub [::router.subs/active-page])
-        {:keys [client-id redirect-uri]} google
-        _                                (react/useEffect (fn []
-                                                            (log/debug "component-did-mount" query)
-                                                            (when-let [code (:code query)]
-                                                              (case (-> query :auth keyword)
-                                                                :google
-                                                                (>evt [:splash/send-google-verification-code code redirect-uri])
-                                                                nil))))
-        classes                          (use-styles)]
+  (let [{:keys [google root-url]}               (<sub [::subs/config])
+        {{:keys [code token] :as query} :query} (<sub [::router.subs/active-page])
+        {:keys [client-id redirect-uri]}        google
+        email?                                  (fn [email]
+                                                  (and (string? email) (re-matches email-pattern email)))
+        [email setEmail]                        (react/useState nil)
```

Comment:
Same for this, I think we are using reagent atoms everywhere for local state instead of react hooks

Reply:
refactored